CHANGELOG.md

```diff
@@ -20,6 +20,11 @@ period. Reserved concurrency can be configured with the `archive_api_reserved_co
 terraform variable on the Cumulus module and increased if you are seeing throttling erorrs.
 The default reserved concurrency value is 8.
 
+### Notable changes
+
+- `cmr_custom_host` variable for `cumulus` module can now be used to configure Cumulus to
```

Comment:
Should this mention that now you *must* include the protocol?  And I assume you didn't rename to cmr_custom_protocol_host to prevent something breaking?

Reply:
Yeah good point. I'll update the note here. And yeah I didn't want to update the variable name so that we could avoid people having to change that even if it is a very minor breaking change.